### PR TITLE
[T] HdGalleryMedia tests

### PR DESCRIPTION
**Pull request by drapisarda**
https://homeday.atlassian.net/browse/FET-47

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -45,6 +45,7 @@
     "vue-zoomer": "homeday-de/vue-zoomer#0.5.0"
   },
   "devDependencies": {
+    "@babel/helper-compilation-targets": "^7.10.4",
```
Comment (viniciuskneves) [inline package.json:48]: What is the reason for this package? I didn't get it.

Comment (drapisarda) [reply package.json:48]: It was having this error https://stackoverflow.com/questions/62246824/error-err-package-path-not-exported-no-exports-main-resolved-in-app-node-m

Comment (viniciuskneves) [reply package.json:48]: When are you getting it? Which command?

File: tests/unit/components/gallery/HDGalleryMedia.spec.js [hunk 2]
```diff
@@ -8,10 +8,26 @@ const wrapperBuilder = wrapperFactoryBuilder(HdGalleryMedia, {
...
     const wrapper = wrapperBuilder();
 
     expect(wrapper.html()).toMatchSnapshot();
   });
+
+  it('renders empty element', () => {
+    const wrapper = emptyWrapperBuilder();
+
+    expect(wrapper.html()).toMatchSnapshot();
+  });
```
Comment (viniciuskneves) [inline tests/unit/components/gallery/HDGalleryMedia.spec.js:24]: I would challenge it as well, maybe we can tackle it differently.

Does it make sense to have this component without an item? Shall we go in the direction of having it with required props?

Comment (drapisarda) [reply tests/unit/components/gallery/HDGalleryMedia.spec.js:24]: Is there to cover 
`default: () => ({}),`

Comment (drapisarda) [reply tests/unit/components/gallery/HDGalleryMedia.spec.js:24]: It does not make much sense is just a little more resilient. 

That test is there to cover HdGalleryMedia:38

Comment (viniciuskneves) [reply tests/unit/components/gallery/HDGalleryMedia.spec.js:24]: Yep, I get it that. My only question is if it makes sense. If does, I'm fine but I just don't see it being used like that.

The only place where `HdGalleryMedia` is used is inside `HdGallery` which has a handler for [empty scenario](https://github.com/homeday-de/homeday-blocks/blob/develop/src/components/gallery/HdGallery.vue#L9).

Does it make sense to keep it with a default of `{}`?
IMHO I think the property should be required but I don't know if we have an use case for empty state on this component.

What I'm challenging here is not the implementation itself, it the reasoning behind it.

Comment (drapisarda) [reply tests/unit/components/gallery/HDGalleryMedia.spec.js:24]: I do agree with you. I'll give it a refactor time

**Comment by drapisarda** [comment]
done in https://github.com/homeday-de/homeday-blocks/pull/523

**Pull request closed without merging**